### ISSUE TITLE
Возможность исключения из бэкапа каталогов symlink в PGDATA

### ISSUE THREAD

**Issue by triwada** [postgrespro/pg_probackup#380]
На текущий момент pg_probackup бэкапит каталоги symlink из PGDATA, но есть необходимость иметь возможность исключать такие каталоги из бэкапа. 
Не все symlink (по понятным причинам pg_wal и pg_log как symlink довольно часто применяется в практике), а только те, что задаст пользователь. По сути альтернатива опции -E external-directories-paths только для указания исключения каталогов.

Т.к. может возникнуть вопрос: зачем в PGDATA symlink на сторонние каталоги, то добавлю такой кейс: в некоторых случаях загрузки/обработки файлов через SQL можно использовать относительные пути (от PGDATA), но для большого числа файлов (и их объёма) приходится их выносить на другой раздел и делать symlink в PGDATA.

**Comment by gsmol** [comment]
Добрый день!
кмк в общем случае эта задача должна решаться через добавление возможности указывать директории (и симлинки на директории), которые не должны попадать в бэкап.
Я правильно понимаю?

**Comment by triwada** [comment]
да, думаю, что нужен параметр типа --exclude-paths PATHNAME (в принципе неважно, чем будет являться значение: действительным файлом или symlink)
